src/Data/List/Relation/Binary/Sublist/Setoid/Properties.agda

```diff
@@ -6,13 +6,14 @@
 
 {-# OPTIONS --cubical-compatible --safe #-}
 
-open import Relation.Binary using (Setoid; _⇒_; _Preserves_⟶_)
+open import Relation.Binary using (Rel; Setoid; _⇒_; _Preserves_⟶_)
 
 module Data.List.Relation.Binary.Sublist.Setoid.Properties
   {c ℓ} (S : Setoid c ℓ) where
 
 open import Data.List.Base hiding (_∷ʳ_)
 open import Data.List.Relation.Unary.Any using (Any)
+open import Data.List.Relation.Binary.Sublist.Heterogeneous using () renaming (minimum to []⊆_)
```

Comment:
Hmm best to not rename things unnecessarily? If you wanted to you could add an extra lemma named `⊆-minimum` here?

Reply:
OK, I'll change, even though it makes the proof less readable, because `[]⊆ xs` is unambiguous for `[] ⊆ xs`, while `minimum xs` would (in my intuitition) mean to extract the minimal element from list `xs` (see Haskell's `Foldable`s).